INSTALLATION.md

```diff
@@ -1,259 +1,250 @@
-1. **Using a mongodb database installed on your local system natively:**
-    1. Follow the setup guide on [mongodb docs](https://www.mongodb.com/docs/manual/administration/install-community/) for your respective operating system.
-1. **Using a containerized mongodb docker container on your local system:**
-    1. Follow the setup guide using this [video tutorial](https://www.youtube.com/watch?v=uklyCSKQ1Po).
+## Setting up MONGODB_URL in .env file
 
-**Note:** If you are running mongodb on a remote system, either a cloud service or a system under your control, then ensure you have provided the correct access permissions and firewall openings for the VM/server/service where the mongodb is hosted.
+Which approach you choose to set up your mongodb database does not matter. What matters is the connection string to that database using which talawa-api can connect to it. Connection string can differ depending on the approach you used to set up your database instance. Please read the official [mongodb docs](https://www.mongodb.com/docs/manual/reference/connection-string/) on connection string. Copy/paste this connection string to the variable named `MONGO_DB_URL` in the `.env` file.
 
-#### The MongoDB Connection String URL
+# Google/firebase
 
-After selecting your hosting method, you will need to:
+You need to have a google account to follow the following steps.
 
-1. Save the MongoDB connection string to be used in the `Talawa-API` `.env` configuration file.
-1. Assign the connection string URL to the environment variable named `MONGO_DB_URL` inside the `.env` file. 
+## Setting up RECAPTCHA_SECRET_KEY in .env file
 
-### Configuring Access Tokens
+We use reCAPTCHA for authentication. Follow these steps:-
 
-Tokens are required for the mobile app to communicate with the API
+1. Visit [this](https://www.google.com/recaptcha/admin/create) url.
+2. Fill in the input blocks as shown in the screenshot:-
 
-1.  Generate an `ACCESS_TOKEN_SECRET` using the `openessl` command below and copy the result to the `ACCESS_TOKEN_SECRET` section of the `.env` file.
+   ![Set up recaptcha page](./image/recaptcha_set_up.webp)
 
-    `openssl rand -hex 32`
+3. Click on `Submit` button.
+4. Copy the generated secret key to variable named `RECAPTCHA_SECRET_KEY` in the `.env` file.
 
-1.  Generate an `REFRESH_TOKEN_SECRET` using the `openessl` command below and copy the result to the `REFRESH_TOKEN_SECRET` section of the `.env` file.
+   ![Set up recaptcha page](./image/recaptcha_secret.webp)
 
-    `openssl rand -hex 32`
+## Setting up MAIL_USERNAME/MAIL_PASSWORD in .env file
 
-### Configuring reCAPTCHA
+**Note:-** Your google account needs to have two factor authentication set up for the following steps to work.
```

Comment:
Please address this `#talawa` Slack post here. Do we really need reCAPTCHA, and if so, should we fix this problem as an issue?

![image](https://user-images.githubusercontent.com/16875803/212991170-811b9f45-2091-4a53-923c-cbdcce3b6765.png)